src/Stratis.StratisD/Program.cs

```diff
@@ -28,14 +34,18 @@ public static async Task Main(string[] args)
                     MinProtocolVersion = ProtocolVersion.ALT_PROTOCOL_VERSION
                 };
 
+                var apiSettings = new ApiSettings(nodeSettings, (s) => {
+                    s.ApiPort = nodeSettings.Network.IsTest() ? TestStratisApiPort : DefaultStratisApiPort;
```

Comment:
I think using `IsTest()` in place of `IsBitcoin` doesn't fix the larger issue - it just replaces it with the same problem...

Reply:
If we can get the new `NetworkType` from NBitcoin this might go away though...

Reply:
`IsTest` is somewhere further down on my list... ;)

Reply:
> If we can get the new `NetworkType` from NBitcoin this might go away though...

Yes, that is another way of doing it.